pkg/operator/operator.go

```diff
@@ -1,5 +1,3 @@
-// Copyright 2022 Google LLC
```

Comment:
did the tool mean to remove the copyright?

Reply:
I think that was probably just me fat fingering the deletion of the line. Good catch. I'll add that back.